### ISSUE TITLE
Latest version fails to connect to chatgpt models

### ISSUE THREAD

**Issue by d3287t328** [unconv/gpt-autopilot#3]
I have tried a few of the latest models and all I get is a Waiting for ChatGPT message. I am able to use the api with other apps and api calls. 

```
Waiting for ChatGPT...
^CERROR in OpenAI request... Trying again
^CTraceback (most recent call last):
  File "/var/home/a/gpt-autopilot/chatgpt.py", line 31, in send_message
    response = openai.ChatCompletion.create(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

using ```Python 3.11.3 (main, May 24 2023, 00:00:00) [GCC 13.1.1 20230511 (Red Hat 13.1.1-2)] on linux```

installed pip requirements, updated latest openai library version, and I am using config.json for model version.

**Comment by unconv** [comment]
Oh, and if an older version works for you, which one? (which commit since there's no versioning right now)

**Comment by unconv** [comment]
Ahh, is it repeating "Waiting for ChatGPT"? That means you have the wrong model in the settings. Make sure that you are using the 0613 model that supports function calling.

This is caused by me forgetting to add an `else: raise` to the `if` at `chatgpt.py` line 41. I will fix this in the next update.

**Comment by d3287t328** [comment]
Run the script through an error checking enhancement in chatgpt4 it will probably catch some good ones!

**Comment by unconv** [comment]
This should now be fixed in #4
